### PR TITLE
add navigation and links

### PR DESCRIPTION
**Pull request by sashameniakin**
added navigation and structure of the project

### CODE REVIEW

File: components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+`;
+const StyledDiv4 = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 20px;
+  width: 46px;
+  height: 21px;
+  background-color: ${props =>
+    props.active === "/profile" ? "rgba(255, 123, 137, 0.5)" : ""};
```
Comment (actyralf) [inline components/Navbar.js:118]: To have an individual StyledComponent for every link is very repetitive and does not scale well. Please try to refactor this by evaluating the pathname match in your `Navbar` component and pass a boolean `active` prop to the `StyledDiv`.

**Review by actyralf — CHANGES_REQUESTED** [review]
Very well done!
Please look into my comment below to improve scalability and reduce code repetition.

File: components/Navbar.js (hunk 1 above)
Comment (sashameniakin) [reply components/Navbar.js:118]: Thank you for the comment! Done.

**Review by actyralf — APPROVED** [review]
Perfect!

**Pull request merged**